Directory.Packages.props

```diff
@@ -0,0 +1,16 @@
+<Project>
+
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
```

Comment:
Can you tell me what these two new properties do? I suspect this is what you meant in your comment in my Bcl.HashCode PR that it was one of the things you wanted us to do first.

Reply:
https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management#transitive-pinning